File: UnitTestProject1/Tests/TestBase.cs
using NUnit.Framework;

namespace UnitTestProject1
{
    public class TestBase
    {
        protected AppManager app;

        [SetUp]
        public void SetupTest()
        {
            app = new AppManager();
        }

        [TearDown]
        public void TeardownTest()
        {
            app.Stop();
        }
    }
}

File: src/test/java/ru/itis/todoist/tests/TestBase.java
package ru.itis.todoist.tests;

import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.BeforeEach;
import ru.itis.todoist.AppManager;

public class TestBase {
    protected static AppManager app;

    @BeforeAll
    public static void setUp() {
        app = AppManager.getInstance();
    }

    @BeforeEach
    public void openHomePage() {
        app.getNavigation().goToHomePage();
    }

    @AfterAll
    public static void tearDown() {
        if (app != null) {
            app.stop();
        }
    }
}
